SQF/dayz_code/configVariables.sqf

```diff
@@ -39,6 +39,10 @@ MaxAmmoBoxes = 3; // Max number of random Supply_Crate_DZE to spawn around the m
 MaxMineVeins = 50; // Max number of random mine veins to spawn around the map
 timezoneswitch = 0; // Changes murderMenu times with this offset in hours.
 DZE_SafeZonePosArray = []; //Prevent players in safeZones from being killed if their vehicle is destroyed. Format is [[[3D POS], RADIUS],[[3D POS], RADIUS]]; Ex. DZE_SafeZonePosArray = [[[6325.6772,7807.7412,0],150],[[4063.4226,11664.19,0],150]];
+DZE_SafeZoneNoBuildItems = []; // Array of object class names not allowed to build near the safe zones listed above. i.e ["VaultStorageLocked","LockboxStorageLocked","Plastic_Pole_EP1_DZ"] etc.
+DZE_SafeZoneNoBuildNear = []; //Array of object class names that are blacklisted to build near. i.e ["Land_Mil_ControlTower","Land_SS_hangar"] etc.
+DZE_SafeZoneNoBuildDistance = 1000; // Distance from safe zones listed above to disallow building near.
+DZE_BlacklistNoBuildDistance = 1000; // Distance from blacklisted objects to disallow building near.
```

Comment:
It may be easier just to whitelist objects, rather than blacklisting them as well. 500 m is probably reasonable as a default, 1 Km is too high, I agree. We may just want to leave it as the size of trader areas by default, otherwise it may cause issues on smaller maps like namalsk. Also we don't even have safezones in the mod, so as eBay notes above, the name doesn't make sense